python/dolfinx/wrappers/fem.cpp

```diff
@@ -775,8 +793,17 @@ void fem(py::module& m)
           },
           py::arg("cell"))
       .def_property_readonly("bs", &dolfinx::fem::DofMap::bs)
-      .def("list", &dolfinx::fem::DofMap::list,
-           py::return_value_policy::reference_internal);
+      .def(
+          "list",
+          [](const dolfinx::fem::DofMap& self)
+          {
+            auto dofs = self.list();
+            std::array<py::ssize_t, 2> shape
+                = {py::ssize_t(dofs.extent(0)), py::ssize_t(dofs.extent(1))};
+            return py::array_t<std::int32_t>(shape, dofs.data_handle(),
```

Comment:
Shouldn't this be const, as discussed in https://github.com/FEniCS/dolfinx/pull/2629 ?

Reply:
Maybe, #2629 doesn't offer a solution yet.

Reply:
What do you think that PR is missing?

Reply:
> What do you think that PR is missing?

You changed it ;).